lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -1,5 +1,11 @@
 module GapIntelligence
   class MerchantPricingTrend < Record
-    attributes :values, :headers
+    def variants
+      @variants ||= begin
+        parsed = MultiJson.load(raw[:response_body]) rescue {}
```

Comment:
As I understand the idea, to improve performance it was updated to eliminate parsing response and return raw body. if we call `MultiJson.load` here I think it should again slow down. Could you please clarify how all this works, because I feel I missed something.